lib/TelnyxMethod.js

```diff
@@ -44,4 +45,20 @@ function telnyxMethod(spec) {
   };
 }
 
+/**
+ * Nested resources related method to populate nest method URL with matching resource object attributes.
+ * e.g.: use `call_control_id` attribute from `call`.
+ * That if exists in call object or in `call.create()` response data
+ */
```

Comment:
JSDoc comment is a bit confusing, if I'm understanding correctly this method just replaces empty `paramsValues` with values set on the parent resource?  Maybe:
> Populate nested method URL params with resource object attributes that match the param name.
> This allows you to do things like setting the `call_control_id` attribute from an existing `call` object on a new instance of `telnyx.calls`.

Reply:
good call! I'll update the doc